src/include/migraphx/op/nonmaxsuppression.hpp

```diff
@@ -38,6 +38,7 @@
 #include <migraphx/check_shapes.hpp>
 #include <migraphx/output_iterator.hpp>
 #include <migraphx/argument.hpp>
+#include <migraphx/par.hpp>
```

Comment:
Totally optional. Please cut-n-paste the documentation from the onnx spec for NMS. Example: please see Unique Operator. Thanks.

Reply:
Done